presto-main/src/main/java/io/prestosql/sql/analyzer/Scope.java

```diff
@@ -73,6 +73,17 @@ private Scope(
         this.namedQueries = ImmutableMap.copyOf(requireNonNull(namedQueries, "namedQueries is null"));
     }
 
+    public Scope findQueryBoundaryScope()
```

Comment:
maybe this method should be called `getQueryBoundaryScope` to have consistent naming with other methods here?